src/Multiple/SmallMultipleHTMLFormatter.php

```diff
@@ -4,25 +4,69 @@
 
 namespace CultuurNet\CalendarSummaryV3\Multiple;
 
+use CultuurNet\CalendarSummaryV3\DateComparison;
+use CultuurNet\CalendarSummaryV3\DateFormatter;
+use CultuurNet\CalendarSummaryV3\HtmlAvailabilityFormatter;
 use CultuurNet\CalendarSummaryV3\Offer\Offer;
-use CultuurNet\CalendarSummaryV3\Periodic\MediumPeriodicHTMLFormatter;
+use CultuurNet\CalendarSummaryV3\RelativeDateHTMLFormatter;
 use CultuurNet\CalendarSummaryV3\Translator;
+use DateTimeInterface;
 
 final class SmallMultipleHTMLFormatter implements MultipleFormatterInterface
 {
+    use RelativeDateHTMLFormatter;
+    /**
+     * @var DateFormatter
+     */
+    private $formatter;
+
     /**
      * @var Translator
      */
     private $translator;
 
     public function __construct(Translator $translator)
     {
+        $this->formatter = new DateFormatter($translator->getLocale());
         $this->translator = $translator;
     }
 
     public function format(Offer $offer): string
     {
-        $formatter = new MediumPeriodicHTMLFormatter($this->translator);
-        return $formatter->format($offer);
+        $dateFrom = $offer->getStartDate()->setTimezone(new \DateTimeZone(date_default_timezone_get()));
```

Comment:
Can these 5 lines be moved to a function? That function could also be used to create the to-date.